### PR TITLE
Add more tests for pair

### PR DESCRIPTION
**Pull request by gohyongjing**
* Add tests for `JsonPairStorage`, `JsonSerialisablePair` and `AddPairCommand`.

### CODE REVIEW

File: src/main/java/seedu/address/model/FriendlyLink.java [hunk 1]
```diff
@@ -208,7 +208,7 @@ public boolean hasPair(Pair pair) {
     }
 
     /**
-     * Adds a {@pair} to {@code FriendlyLink}.
+     * Adds a {code @pair} to {@code FriendlyLink}.
```
Comment (wz2k) [inline src/main/java/seedu/address/model/FriendlyLink.java:211]: There is a small typo here.
```suggestion
     * Adds a {@code pair} to {@code FriendlyLink}.
```

File: src/main/java/seedu/address/model/FriendlyLink.java [hunk 2]
```diff
@@ -217,6 +217,18 @@ public void addPair(Pair pair) {
         pairs.add(pair);
     }
 
+    /**
+     * Adds a pair consisting of elderly with {@code elderlyNric} and volunteer with {@code volunteerNric}
+     * from {@code FriendlyLink}.
```
Comment (wz2k) [inline src/main/java/seedu/address/model/FriendlyLink.java:222]: Small typo.
```suggestion
     * to {@code FriendlyLink}.
```

File: src/main/java/seedu/address/model/Model.java [hunk 3]
```diff
@@ -123,6 +123,19 @@ public interface Model {
      */
     void setVolunteer(Volunteer target, Volunteer editedVolunteer);
 
+    /**
+     * Adds the given pair.
+     * {@code pair} must not already exist in the address book.
```
Comment (wz2k) [inline src/main/java/seedu/address/model/Model.java:128]:
```suggestion
     * {@code pair} must not already exist in FriendlyLink.
```

File: src/main/java/seedu/address/model/Model.java [hunk 4]
```diff
@@ -123,6 +123,19 @@ public interface Model {
...
+     */
+    void addPair(Pair pair);
+
+    /**
+     * Adds the given pair.
+     * The pair consisting of elderly with {@code elderlyNric} and volunteer with {@code volunteerNric}
+     * must not already exist in the address book.
```
Comment (wz2k) [inline src/main/java/seedu/address/model/Model.java:135]:
```suggestion
     * must not already exist in FriendlyLink.
```

**Review by wz2k — APPROVED** [review]
LGTM, just a few typos that needs to be fixed.

File: src/main/java/seedu/address/model/Model.java (hunk 4 above)
Comment (gohyongjing) [reply src/main/java/seedu/address/model/Model.java]: Resolved in commit 71e90f4f6ec2f74d637d12a4c6f48d26cb9ecb87

File: src/main/java/seedu/address/model/Model.java (hunk 3 above)
Comment (gohyongjing) [reply src/main/java/seedu/address/model/Model.java]: Resolved in commit 71e90f4f6ec2f74d637d12a4c6f48d26cb9ecb87

File: src/main/java/seedu/address/model/FriendlyLink.java (hunk 2 above)
Comment (gohyongjing) [reply src/main/java/seedu/address/model/FriendlyLink.java]: Resolved in commit 71e90f4f6ec2f74d637d12a4c6f48d26cb9ecb87

File: src/main/java/seedu/address/model/FriendlyLink.java (hunk 1 above)
Comment (gohyongjing) [reply src/main/java/seedu/address/model/FriendlyLink.java]: Resolved in commit 71e90f4f6ec2f74d637d12a4c6f48d26cb9ecb87

**Comment by gohyongjing** [comment]
Typos have been resolved @wz2k

File: src/main/java/seedu/address/model/ModelManager.java [hunk 5]
```diff
@@ -162,6 +162,18 @@ public boolean hasPair(Pair pair) {
         return friendlyLink.hasPair(pair);
     }
 
+    @Override
+    public void addPair(Nric elderlyNric, Nric volunteerNric) {
+        friendlyLink.addPair(elderlyNric, volunteerNric);
+        // TODO: implement updateFilteredPersonList(PREDICATE_SHOW_ALL_PAIRS);
```
Comment (Zxun2) [inline src/main/java/seedu/address/model/ModelManager.java:168]: Would it be too troublesome to add this to this PR? The function is already implemented, right?

File: src/main/java/seedu/address/model/ModelManager.java [hunk 6]
```diff
@@ -162,6 +162,18 @@ public boolean hasPair(Pair pair) {
...
+    }
+
+    @Override
+    public void addPair(Pair pair) {
+        friendlyLink.addPair(pair);
+        // TODO: implement updateFilteredPersonList(PREDICATE_SHOW_ALL_PAIRS);
```
Comment (Zxun2) [inline src/main/java/seedu/address/model/ModelManager.java:174]: Same here

**Review by Zxun2 — COMMENTED** [review]
Quick question 👇🏻

File: src/main/java/seedu/address/model/ModelManager.java (hunk 5 above)
Comment (gohyongjing) [reply src/main/java/seedu/address/model/ModelManager.java:168]: lets put this in another pull request to keep this PR small

**Pull request merged**